app/assets/javascripts/components/widgets_box.js.jsx

```diff
@@ -19,8 +19,7 @@ var WidgetsBox = React.createClass({
   render() {
     return (
       <div className='widgets'>
-        <Widgets widgets={this.state.data} campaign_page_id={ this.props.campaign_page_id } />
-        <WidgetTextForm onWidgetSubmit={this.handleWidgetSubmit} page_id={ this.props.campaign_page_id }/>
```

Comment:
Moved this reference deeper into the composition so that forms exist as part of the widget.